tests/unit/tasks/server/livereload-server-test.js

```diff
@@ -75,6 +75,16 @@ describe('livereload-server', function() {
           preexistingServer.close(done);
         });
     });
+
+    it('starts with custom host', function() {
+      return subject.start({
+        liveReloadHost: '127.0.0.1',
```

Comment:
but we already showed localhost, so i wanted there to be some visual change in the text since 0.0.0.0 gets transformed to localhost already.